stable/sonatype-nexus/values.yaml

```diff
@@ -1,68 +1,85 @@
-# Default values for nexus.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
 replicaCount: 1
-image:
-  repository: clearent/nexus
-  tag: 3.5.1-02
 
-# node labels for node selection
-# nodeSelector:
-#   cloud.google.com/gke-nodepool: default-pool
-
-service:
-  name: nexus
-  type: LoadBalancer
-  externalPort: 8081
+nexus:
+  imageName: quay.io/travelaudience/docker-nexus
```

Comment:
You changed the image from some random image to another random image. Why don't we use the official image?

https://hub.docker.com/r/sonatype/nexus3/

Reply:
The official image makes it difficult to provide restore capabilities, and the Nexus service needs to be stopped in order for them to be applied. 

This is better documented here: https://github.com/travelaudience/kubernetes-nexus#restore